Training/2.0 Libraries.md

```diff
@@ -52,14 +59,18 @@ Normally we would want to be able to extract the information from tables directl
 [Training2_example.csv](https://github.com/alepgr/gubmes-hda/blob/main/Training/Training2_example.csv)
 
 ```
-
+data = pd.read_csv('Training2_example.csv')
+df = pd.DataFrame(data)
+df.head()
 ```
 
 ### Task 4a)
 Using your DataFrame from Task 4a), calculate the average systolic blood pressure of Dementia patients.
 
 ```
-
+a = df[df.Conditions == Dementia]
+b = a.df['Systolic Blood Pressure']
+mean = b.mean()
```

Comment:
Excellent attempt! Your method is correct but there's a couple of syntax errors:
1) The column name in the data frame is called Condition not Conditions. It's a tiny difference, but for python you have to be precise.
2) The value Dementia is a string so it must be enclosed by speech marks or inverted commas. So correct line 71 would be: `a = df[df.Condition == 'Dementia']`
3) a is already a data frame, so the correct way to filter a data frame is: `b = a['Systolic Blood Pressure']`